.circleci/config.yml

```diff
@@ -103,3 +113,5 @@ workflows:
           filters:
             branches:
               only: master
+            tags:
+              only: /.*/
```

Comment:
I think you need to push to catalogs and collections too.

Reply:
There's no chart in this project, this is just a sidecar container for g8s-prometheus.